plugins/sample/README.md

```diff
@@ -0,0 +1,41 @@
+# sample_plugin
```

Comment:
This file is showing up as a new file rather than a modification of the old README because it both moved, and changed substantially due to moving a bunch of the specific instructions into comments in the code.